SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorPackageAssembly.cpp

```diff
@@ -149,19 +151,43 @@ void FSpatialGDKPackageAssembly::OnTaskCompleted(int32 TaskResult)
 		{
 			FString NotificationMessage = FString::Printf(TEXT("Failed assembly upload to project: %s"), *FSpatialGDKServicesModule::GetProjectName());
 			ShowTaskEndedNotification(NotificationMessage, SNotificationItem::CS_Fail);
+			if (Status == EPackageAssemblyStatus::ASSEMBLY_EXISTS)
+			{
+				FMessageDialog::Open(EAppMsgType::Ok, LOCTEXT("Assembly_Exists", "The assembly has previously been uploaded enable the 'Force Overwrite on Upload' option in the Cloud Deployment dialog to overwrite the existing assembly."));
```

Comment:
Can we present a user with a Yes/No dialog to overwrite it without restarting the process?

Reply:
At first I thought, sure, but on second thought what if the user wants to correct this by renaming the assembly? I feel that having a link to open the Cloud Deployment dialog maybe the best option. With regard to how we can avoid forcing the user to regen and rebuild prior to uploading the assembly, I think we should take a step back and look at that in the larger context: [UNR-3546](https://improbableio.atlassian.net/browse/UNR-3546) I've added this ticket to capture what I feel is a more comprehensive solution.